src/website/src/app/documentation/demos/timeline/timeline.demo.html

```diff
@@ -335,18 +326,21 @@ <h4>Examples</h4>
       <p>Lets look at some of the elements in isolation and then put everything together at the bottom of the page.</p>
 
 <!--      includes vertical and horizontal code snippets-->
-      <h3> Timeline</h3>
+      <h3>Timeline</h3>
```

Comment:
I have a generic ask here, I'd rather put the Angular component first and then cover the HTML version at the bottom. Can you reorganize this and see if you can make it more clear? Its a bit confusing the way things are laid out right now.

Reply:
@gnomeontherun  I'm confused to about the structure,

Unfortunately I cannot find a common pattern in other places; for instance in [Tabs](https://v2.clarity.design/tabs) the static version is before the Angular version while in [Input](https://v2.clarity.design/input) the Angular one is the first.

My personal opinion is that, since most part of the Job is done in CSS (the angular component is simply a wrapper), CSS (static) should go first; then in Angular description you see only the differences with the static one.

My Idea would be a structure like
```
- Design Guidelines
- Code & Examples
  - Structure
  - CSS classes
     - Full Demo (horizontal and vertical)
   - Angular Components
     - Full Demo (horizontal and vertical)
```

I can swap the Static and the angular one but in that case I should also duplicate/rewrite information about the icons

Before to start I'd like to agree on the structure